src/Sylius/Behat/Page/Shop/Account/Order/IndexPage.php

```diff
@@ -86,14 +69,16 @@ public function isItPossibleToChangePaymentMethodForOrder(OrderInterface $order)
         return $row->hasLink('Pay');
     }
 
-    /**
-     * {@inheritdoc}
-     */
+    public function openLastOrderPage(): void
+    {
+        $this->getElement('last_order')->click();
+    }
+
     protected function getDefinedElements(): array
     {
         return array_merge(parent::getDefinedElements(), [
-            'customer_orders' => 'table',
-            'last_order' => 'table tbody tr:last-child a:contains("Show")',
+            'customer_orders' => '[data-test-grid-table]',
+            'last_order' => '[data-test-grid-table-body] tr:last-child a:contains("Show")',
```

Comment:
/cc @kulczy can we do it better?

Reply:
we can find the last element in the loop and add a new attribute, it seems a bit better solution

Reply:
Maybe we should add `{ sylius_test_html_attribute('grid-table-row') }` in: https://github.com/Sylius/Sylius/blob/ae812d3fc91f34fae353e106e68c65a70c089f80/src/Sylius/Bundle/UiBundle/Resources/views/Macro/table.html.twig#L25 and replace `tr` to `['data-test-grid-table-row']`
It would be more consistent with other changes